### ISSUE TITLE
[AVT1] IBM 2.5.3 Label in Name violation in DataTable with Toolbar

### ISSUE THREAD

**Issue by joshblack** [carbon-design-system/carbon#4861]
When running DAP on our storybook, we run into a violation of IBM 2.5.3 Label in Name:

![image](https://user-images.githubusercontent.com/3901764/70647850-7e767100-1c0f-11ea-873d-d536e0a8fdc0.png)

I came across this violation while working on #3743
